### DISCUSSION TITLE
Question regarding to invoices and discounts

### DISCUSSION THREAD

**Discussion by LiaraAlis** [kimai/kimai#3904] [Questions & Answers]
I also have this problem and bought the expenses plugin recently.
Now, I've seen a free plugin for deducation times (https://www.kimai.org/store/keleo-deduction-time-bundle.html). What exactly is the difference between the expenses plugin and this free deducation time bundle plugin in relation to discounts for invoices?
Am I right that the deducation plugin only adds the availability to make timesheets negative while the expenses plugin adds a completely new entity for that?

In my case, for some customers I arrange a fixed budget for project with my customers and I guarantee to not charge more as agreed. Would it be possible to support fixed values instead of time for projects?

And is it already clear when the Expenses plugin for v2 will be released?

**Comment by kevinpapst** [comment]
> What exactly is the difference between the expenses plugin and this free deducation time bundle plugin in relation to discounts for invoices?

I don't see any connection between them. Deduction has exactly one very limited use-case: to turn times into negative times.

> Would it be possible to support fixed values instead of time for projects?

In that case you usually would create an Expense for the one-time value.
And then turn store all time entries as non-billable, so they are only used for statistical purpose.
